static/script-tests/tests/devices/mediaplayer/cehtmlcommontests.js

```diff
@@ -1342,6 +1372,24 @@ window.commonTests.mediaPlayer.cehtml.mixinTests = function (testCase, mediaPlay
         });
     };
 
+    mixins.testNoSeekSentinelActivatedWhenCurrentTimeIsReportedAsZeroDuringPlayback = function(queue) {
+        expectAsserts(1);
+        var self = this;
+        runMediaPlayerTest(this, queue, function (MediaPlayer) {
+            var SEEK_SENTINEL_TOLERANCE = 15;
+            getToPlaying(this, MediaPlayer, SEEK_SENTINEL_TOLERANCE + 20);
+
+            var eventHandler = this.sandbox.stub();
+            this._mediaPlayer.addEventCallback(null, eventHandler);
+
+            for (var i=0; i<4; i++) {
+                fakeCEHTMLObject.playPosition = 0;
```

Comment:
Would it make more sense to fire the sentinels 3 times, then set the playPosition to 0 before firing the sentinels 1 last time? You could then assert that the seek sentinel wasn't called at all instead of exactly twice, which would match the name of the test closer.